src/drivers/actuators/vertiq_io/vertiq_client_manager.cpp

```diff
@@ -0,0 +1,237 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2012-2022 PX4 Development Team. All rights reserved.
```

Comment:
Do we need to update this license header at all? The years at least? Not really sure what the PX4 policy on this is, but I would imagine they would want code to come in with its current year

Reply:
their dev documentation (https://docs.px4.io/main/en/modules/hello_sky.html) still lists 2022

Reply:
They wrote that dev documentation in 2022, but you wrote this file in 2024.

If we look at some of the files in PX4, we see they don't just use 2022. Here's one with 2023: 
https://github.com/PX4/PX4-Autopilot/blob/main/boards/px4/fmu-v6xrt/src/CMakeLists.txt

Here's one with 2024:
https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/ekf2/EKF/velocity_fusion.cpp

They just haven't updated their example to show the current year, I think that's implied